### ISSUE TITLE
Add 404 Page

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/indokhaadyam#4]
## Directions

- This should be a page that is loaded whenever a user looks up any resource (page) which does not exist
- It should have a "Go to Home" button in the center
- Be creative and add some animation/illustration which is consistent with the color theme and the same time resembles the project's idea revolving around food.
- [Sample Reference](https://portfolio.smaranjitghose.com/no)
- Feel free to take inspiration from website like Codepen but refrain from copying it exactly!

**Comment by bansalshubh** [comment]
@smaranjitghose , can I work on this issue?

**Comment by Geeky-Sam01** [comment]
Please, Assign it to me.

**Comment by smaranjitghose** [comment]
Sure @bansalshubh . 
@Geeky-Sam01  You may give a follow-up PR (perhaps a design update) once @bansalshubh is done with a basic 404 page with the instructions given above

**Comment by Mohith234** [comment]
I want to work on this issue under JWOC

**Comment by atulbansal82** [comment]
I want to work on this issue under JWOC

**Comment by raxvab** [comment]
I want to work on this issue under JWOC it's gonna be one of the first opportunities for me to start with open source can i get this issue assigned.

**Comment by Mohith234** [comment]
I want to work on this issue under JWOC

**Comment by r3yc0n1c** [comment]
I would like to work on this issue under JWOC'22

**Comment by Shreyan-12345** [comment]
is the issue completed , if not then i want to contribute @smaranjitghose

**Comment by r3yc0n1c** [comment]
done... creating the PR in a minute
